PostgreSQL ecpg/initdb manual page fixes

This was generated based on automatic Red Hat manual page scan (private
RHBZ#948933).

### ecpg ###

diff -up postgresql-9.2.4/doc/src/sgml/man1/ecpg.1.broken postgresql-9.2.4/doc/src/sgml/man1/ecpg.1
--- postgresql-9.2.4/doc/src/sgml/man1/ecpg.1.broken	2013-06-05 10:54:39.610443837 +0200
+++ postgresql-9.2.4/doc/src/sgml/man1/ecpg.1	2013-06-05 10:59:24.115997253 +0200
@@ -81,6 +81,11 @@ INFORMIX_SE\&.
 Define a C preprocessor symbol\&.
 .RE
 .PP
+\fB\-h \fR
+.RS 4
+Parse a header file, this option includes option \fB\-c\fR\&.
+.RE
+.PP
 \fB\-i\fR
 .RS 4
 Parse system include files as well\&.
@@ -129,6 +134,11 @@ Allow question mark as placeholder for c
 .RE
 .RE
 .PP
+\fB\-\-regression\fR
+.RS 4
+Run in regression testing mode\&.
+.RE
+.PP
 \fB\-t\fR
 .RS 4
 Turn on autocommit of transactions\&. In this mode, each SQL command is automatically committed unless it is inside an explicit transaction block\&. In the default mode, commands are committed only when

### initdb ###

diff -up postgresql-9.2.4/doc/src/sgml/man1/initdb.1.broken postgresql-9.2.4/doc/src/sgml/man1/initdb.1
--- postgresql-9.2.4/doc/src/sgml/man1/initdb.1.broken	2013-06-06 10:26:48.405835762 +0200
+++ postgresql-9.2.4/doc/src/sgml/man1/initdb.1	2013-06-06 10:29:46.084209958 +0200
@@ -227,6 +227,11 @@ determines that an error prevented it fr
 .PP
 Other options:
 .PP
+\fB\-s\fR, \fB\-\-show\fR
+.RS 4
+Print the internal settings, then exit\&.
+.RE
+.PP
 \fB\-V\fR, \fB\-\-version\fR
 .RS 4
 Print the
